examples/basics/ontologies.ipynb

```diff
@@ -85,8 +85,18 @@
             "cell_type": "markdown",
```

Comment:
<p>Do we believe this is going to be particularly useful for customers? Are we finding confusion from customers at this moment about this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Yes, I have seen a few customers trying this out and not knowing the names for certain tools that you must use, and it's not well documented.